src/shared/hooks/use-shutterbug.ts

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from "react";
+import Shutterbug from "shutterbug";
+
+// This hook can be used by any interactive that requires snapshots.
+export const useShutterbug = () => {
+  useEffect(() => {
+    Shutterbug.enable("#app");
```

Comment:
If you pass the selector as an argument (perhaps defaulting to `#app`) the hook would be usable in more contexts.